tools/logging.py

```diff
@@ -8,17 +8,31 @@
 #
 # license: GPLv2
 #
+
+# Standard library imports
 import datetime
-import json
 import os
 import sys
 
+# Third party imports (anything installed into the local Python environment)
+
+# Local application imports (anything from EESSI/eessi-bot-software-layer)
+
+# TODO possibly replace 'HOME' (and log file name) with a configurable value
```

Comment:
Or just one variable to determine full path to log file, with `~/eessi-bot-software-layer.log` being the default is none is specified?

Reply:
Turns out we already have a config setting for this ... or more precisely two:

```ini
[event_handler]
log_path = ...

[job_manager]
log_path = ...
```

We also have an issue to improve logging (#91). Will change the comment accordingly.